src/DevChatter.DevStreams.Web/Areas/Identity/Pages/_ViewImports.cshtml

```diff
@@ -0,0 +1,5 @@
+﻿@using Microsoft.AspNetCore.Identity
+@using DevChatter.DevStreams.Web.Areas.Identity
+@using Microsoft.AspNetCore.Identity
```

Comment:
Am I crazy, or are there two references to `Microsoft.AspNetCore.Identity`?

Reply:
Unfortunately, it did generate the file with duplicate references. I'll fix this.

Reply:
Fixed 👍